pkg/setting/setting_test.go

```diff
@@ -296,6 +296,33 @@ func TestLoadingSettings(t *testing.T) {
 			So(value, ShouldEqual, "")
 		})
 	})
+
+	Convey("Test reading provisioning config", t, func() {
```

Comment:
I thought about your use of `HomePath`, and figure it's probably a better idea to not use the global `HomePath` variable (we want to get rid of the globals in any case). I also think it's wise to use an absolute path instead of "../..", since it can be really helpful when debugging failures (a relative path leaves you wondering what it actually resolves to).

What do you think of using a `homePath` variable as follows in the tests?
```suggestion
	Convey("Test reading provisioning config", t, func() {
			homePath, err := filepath.Abs("../..")
			So(err, ShouldBeNil)

```

Reply:
For me, getting rid of `HomePath` looks a bit off this PR subject. Tell me whether I should do it here or open another PR.

However, I have no objections to correct the test.

Reply:
@nabokihms Sorry, didn't mean for you to get rid of HomePath in general, I was just referring to your tests :) I saw they could be written without using the global Homepath.